src/coord/timestamp.rs

```diff
@@ -653,7 +661,16 @@ impl Timestamper {
             .expect("Failed to execute select statement")
             .query_and_then(params![SqlVal(&id.sid), SqlVal(&id.vid)], |row| -> Result<_, failure::Error> {
                 let pcount: SqlVal<i32> = row.get(0)?;
-                let pid: SqlVal<i32> = row.get(1)?;
+                let pid: SqlVal<PartitionId> = match row.get(1) {
+                    Ok(val) => val,
+                    Err(err) => {
+                        error!("Reading from timestamps and expected type PartitionId, found other: {}", err);
```

Comment:
oh, ditch this error message, I think; or downgrade all the way to a trace. It'll be unbelievably spammy on startup on any old materialized data directory. Potentially hundreds of thousands of messages.